client/src/components/questionsAnswers/Accordion.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import QuestionCard from './QuestionCard.jsx';
+
+function Accordion(props) {
```

Comment:
I recommend to revise your component name to provide additional context to the component and to avoid using CSS design elements as the name.